classes/form/FormBuilderVocabulary.inc.php

```diff
@@ -54,6 +54,8 @@
  *   required: Adds an asterisk and a .required class to the element's label
  */
 
+define('INSERT_TAG_VARIABLE_TYPE_PLAIN_TEXT', 'PLAIN_TEXT');
```

Comment:
Interesting and really helpful.

But do I have to declare it somewhere special, because I don't seem to get the variable at the javascript.